src/core/core.controller.js

```diff
@@ -788,10 +788,16 @@ class Chart {
 		return Interaction.modes.index(this, e, {intersect: false});
 	}
 
-	getElementsAtEventForMode(e, mode, options) {
+	/**
+	 * @param {IEvent} e
+	 * @param {string} mode
+	 * @param {any} options
+	 * @param {boolean} [useFinalPosition]
```

Comment:
It might be helpful to add a description to `useFinalPosition` here since `core.controller` is an entry point for most code paths